ruby/example_code/dynamodb/dynamodb_ruby_example_read_multiple_items.rb

```diff
@@ -0,0 +1,161 @@
+# Here, the FilmDirector, MonthsInTheatre, and AcademyAwardWinnerStatus attributes are returned from the Movies table.
+# The PlayDirector, MonthsInTheatre, and TonyAwardWinnerStatus attributes are returned from the Plays table.
+resp  = client.batch_get_item({
+request_items: {
+    "Movies" => {
+        keys: [
+            {
+                "Year"  => "2015",
+                "MovieTitle" => "The Big New Movie",
+                "Plot" => "Action",
+                "Rating" => "Excellent",
+            },
+            {
+                "Year" => "2009",
+                "MovieTitle" => "A Nature Documentary",
+                "Plot" => "Documentary Short",
+                "Rating" => "Fair",
+            },
+            {
+                "Year" => "1983",
+                "MovieTitle" => "An Animated Fairy Tale",
+                "Plot" => "Children's Fiction",
+                "Rating" => "Good",
+            },
+        ],
+# To parse the response by item, we include the primary key values for the items in our request with the
+# projection_expression parameter
+        projection_expression: "FilmDirector, MonthsInTheatre, AcademyAwardWinnerStatus",
+          },
+        },
+      })
+
+    "Plays" => {
+        keys: [
+            {
+                "Year" => "2019",
+                "PlayTitle" => "King Sear",
+                "Plot" => "Tragedy",
+                "Rating" => "Poor",
+            },
+            {
+                "Year" => "1913",
+                "PlayTitle" => "A Midautumn Night's Dream",
+                "Plot" => "Comedy",
+                "Rating" => "Excellent",
+            },
+        ],
+        projection_expression: "PlayDirector, WeeksInTheatre, TonyAwardWinnerStatus",
+            }
+
+resp.to_h outputs the following:
+{
+    responses: {
+        "Movies" => [
+            {
+                "FilmDirector" => "John Doe",
+                "MonthsInTheatre" => "9",
+                "AcademyAwardWinnerStatus" => "Yes",
+            },
+            {
+                "FilmDirector" => "Jane Doe",
+                "MonthsInTheatre" => "4",
+                "AcademyAwardWinnerStatus" => "No",
+            },
+            {
+                "FilmDirector" => "Mr. Director",
+                "MonthsInTheatre" => "2",
+                "AcademyAwardWinnerStatus" => "Yes",
+            },
+        ],
+        "Plays" => [
+            {
+                "PlayDirector" => "Willy Shakes",
+                "WeeksInTheatre" => "3",
+                "TonyAwardWinnerStatus" => "No",
+            },
+            {
+                "PlayDirector" => "W.E. Shakingspear",
+                "WeeksInTheatre" => "23",
+                "TonyAwardWinnerStatus" => "Yes",
+            },
+        ],
+       }
+
+begin
+# batch_get_item retrieves the attributes of our items from the Movies table and Plays table in parallel to minimize
+# response latency.
+
+def batch_get_item(params = {}, options = {})
```

Comment:
I'm not sure this method makes sense as it is.. typically we don't have customers build and send requests, we would ask them to call `client.batch_get_item` as you did above.